swe/ex2b_MMS.c

```diff
@@ -2117,16 +2117,13 @@ PetscErrorCode AddSourceTerm(RDyApp app, Vec F, PetscReal t) {
 
         PetscReal velocity_MMS = PetscSqrtReal(Square(u_MMS) + Square(v_MMS));
 
-        PetscReal tb_MMS = Cd_MMS * velocity_MMS / h_MMS;
-
-        PetscReal dt         = app->dt;
-        PetscReal factor_MMS = tb_MMS / (1.0 + dt * tb_MMS);
+        PetscReal dt = app->dt;
 
         PetscReal h_MMS1 = h0 * (1 + PetscSinScalar(PI * xc / Lx) * PetscSinScalar(PI * yc / Ly)) * PetscExpScalar((t + dt) / t0);
         PetscReal u_MMS1 = u0 * PetscCosScalar(PI * xc / Lx) * PetscSinScalar(PI * yc / Ly) * PetscExpScalar((t + dt) / t0);
         PetscReal v_MMS1 = v0 * PetscSinScalar(PI * xc / Lx) * PetscCosScalar(PI * yc / Ly) * PetscExpScalar((t + dt) / t0);
```

Comment:
The computation of `<h|u|v>_MMS1` can also be deleted. Right?

Reply:
Yes. Those lines are deleted now.